app/services/platform/submitter_payload.rb

```diff
@@ -37,12 +37,20 @@ def concatenation_with_reference_number(text)
     end
 
     def meta
-      {
-        pdf_heading: concatenation_with_reference_number(ENV['SERVICE_EMAIL_PDF_HEADING']),
-        pdf_subheading: ENV['SERVICE_EMAIL_PDF_SUBHEADING'].to_s,
-        submission_at: Time.zone.now.iso8601,
-        reference_number: user_data['moj_forms_reference_number']
-      }.compact
+      if ENV['SERVICE_EMAIL_OUTPUT'].blank?
+        {
+          pdf_heading: "JSON Submission for #{service.service_name}",
+          pdf_subheading: '',
+          submission_at: Time.zone.now.iso8601
+        }.compact
+      else
```

Comment:
Shouldn't we be able to post to an API and also get email submissions? So rather than `else` this should send the JSON payload if the `SERVICE_OUTPUT_JSON_ENDPOINT` and send the email payload if `SERVICE_EMAIL_OUTPUT` is present?

Reply:
If we have set email confirmation (SERVICE_EMAIL_OUTPUT) and api submission (SERVICE_OUTPUT_JSON_ENDPOINT and SERVICE_OUTPUT_JSON_KEY), then we concatenate the actions in line 56.